libmscore/excerpt.cpp

```diff
@@ -1142,13 +1142,49 @@ QList<Excerpt*> Excerpt::createAllExcerpt(MasterScore *score)
                   for (int i = part->startTrack(), j = 0; i < part->endTrack(); i++, j++) {
                         e->tracks().insert(i, j);
                         }
-                  QString name = createName(part->partName(), all);
+                  QString name = part->partName();
```

Comment:
but the function `ExcerptsDialog::createName(const QString& partName)` calls `Excerpt::createName(const QString& partName, QList<Excerpt*> excerptList)`